src/v/cluster/members_manager.cc

```diff
@@ -53,7 +54,18 @@ members_manager::members_manager(
   , _allocator(allocator)
   , _storage(storage)
   , _as(as)
-  , _rpc_tls_config(config::shard_local_cfg().rpc_server_tls()) {}
+  , _rpc_tls_config(config::shard_local_cfg().rpc_server_tls())
+  , _update_queue(max_updates_queue_size) {
+    auto sub = _as.local().subscribe([this]() noexcept {
+        _update_queue.abort(
+          std::make_exception_ptr(ss::abort_requested_exception{}));
```

Comment:
does the queue need to be aborted when the members manager is stopped?

Reply:
yes, this will break the dependency chain, since members manager is the last to shut down.